components/automate-ui/src/app/pages/team-details/team-details.component.spec.ts

```diff
@@ -76,17 +90,125 @@ describe('TeamDetailsComponent', () => {
     }).compileComponents();
   }));
 
+  const someTeam = <Team>{
+    id: 'some-team',
+    name: 'some team',
+    guid: 'a-team-uuid-01',
+    projects: []
+  };
+
   beforeEach(() => {
+    router = TestBed.get(Router);
+    spyOn(router, 'navigate').and.stub();
+
+    jasmine.addMatchers(customMatchers);
     fixture = TestBed.createComponent(TeamDetailsComponent);
     component = fixture.componentInstance;
-
+    component.team = someTeam;
     fixture.detectChanges();
   });
 
   it('should be created', () => {
     expect(component).toBeTruthy();
   });
 
+  it('defaults to showing users section', () => {
+    expect(component.tabValue).toBe('users');
+  });
+
+  it('show users section when users tab is selected', () => {
+    component.onSelectedTab({ target: { value: 'users' } });
+    expect(component.tabValue).toBe('users');
```

Comment:
Just realized this on my PR and the same on your PR: this test passes, yes, but if you change `users` to `foo` on both L120 and L121 it will still pass, and that feels just slightly bad. Here's a fix:
```
    const tabName: TeamTabNames = 'users';
    component.onSelectedTab({ target: { value: tabName } });
    expect(component.tabValue).toBe(tabName);
```
The first line will throw a compile error if `users` is switched to `foo`.